src/platform/packages/shared/kbn-esql-ast/src/commands/registry/eval/autocomplete.ts

```diff
@@ -20,6 +21,13 @@ import {
 import type { ICommandCallbacks } from '../types';
 import { Location, type ICommandContext, type ISuggestionItem } from '../types';
 
+const FUNCTIONS_TO_IGNORE = {
```

Comment:
We pass a configuration at the command level that lets us decide what to show and what to hide, and allows us to skip this role for certain inner cases.